src/components/CallView/Grid/Grid.vue

```diff
@@ -1011,7 +1017,7 @@ export default {
 }
 
 .stripe--collapse {
-	position: absolute;
+	position: absolute !important;
 	top: calc(-1 * var(--default-clickable-area));
 	right: 0;
 	z-index: 2;
```

Comment:
Could we wrap a button in a new absolute block instead of overriding its styles with `!important`?

Reply:
I don't want to add a new wrapper.
But adding some weight to this rule should be enough for solving, so i would go with a solution like this:


```css
.grid-main-wrapper {
    position: relative;

    & .stripe--collapse {
        position: absolute;
    }
}
```